install_config/monitoring/configuring-openshift-cluster-monitoring.adoc

```diff
@@ -0,0 +1,95 @@
+[[configuring-openshift-cluster-monitoring]]
+= Configuring OpenShift cluster monitoring
+:data-uri:
+:icons:
+:experimental:
+:prewrap!:
+
+The OpenShift Ansible `openshift_cluster_monitoring_operator` role configures and deploys the Cluster Monitoring Operator using the variables from the xref:../install/configuring_inventory_file.adoc#configuring-ansible[inventory file].
+
+.Ansible variables
+[options="header"]
+|===
+
+|Variable |Description
+
+|`openshift_cluster_monitoring_operator_install`
+| Deploy the Cluster Monitoring Operator if `true`. Otherwise, undeploy. This variable is set to `true` by default.
+
+|`openshift_cluster_monitoring_operator_prometheus_storage_capacity`
+| The persistent volume claim size for each of the Prometheus instances. This variable applies only if `openshift_cluster_monitoring_operator_prometheus_storage_enabled` is set to `true`. Defaults to `50Gi`.
+
+|`openshift_cluster_monitoring_operator_alertmanager_storage_capacity`
+| The persistent volume claim size for each of the Alertmanager instances. This variable applies only if `openshift_cluster_monitoring_operator_alertmanager_storage_enabled` is set to `true`. Defaults to `2Gi`.
+
+|`openshift_cluster_monitoring_operator_node_selector`
+| Set to the desired, existing xref:../admin_guide/scheduling/node_selector.adoc#admin-guide-sched-selector[node selector] to ensure that pods are placed onto nodes with specific labels. Defaults to `node-role.kubernetes.io/infra=true`.
+
+|`openshift_cluster_monitoring_operator_alertmanager_config`
+| Configures Alertmanager. See xref:configuring-alertmanager.adoc#configuring-alertmanager[Configuring Alertmanager]
```

Comment:
The link for Configuring Alertmanager is broken.

s/[Configuring Alertmanager]/[Configuring Alertmanager].